src/firmware-version.h

```diff
@@ -0,0 +1,14 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+#pragma once
+
+#include <rsutils/version.h>
+
+
+namespace librealsense {
+
+
+using firmware_version = rsutils::version;
```

Comment:
This can be a problem.
Better to specifically use rsutils::version, 
firmware_version is spread around the code as local var and we always do `using librealsense`

Reply:
I just moved it in this PR. If you want I can work to change but it's a whole lot of changes in many many places.